assets/state/states/qam_button_submenu.tres

```diff
@@ -0,0 +1,8 @@
+[gd_resource type="Resource" script_class="State" load_steps=2 format=3 uid="uid://bfoequ6xb7csn"]
+
+[ext_resource type="Script" path="res://core/systems/state/state.gd" id="1_6k42f"]
+
+[resource]
+script = ExtResource("1_6k42f")
+name = "qam_button_submenu"
+data = null
```

Comment:
I think we can just remove this state and just use `focus_entered` signals to update the "last_child"

Reply:
SAA